README.md

```diff
@@ -269,6 +270,7 @@
 
 **Legend**: 💼 Package · 📟 Command line tool/generator · 🚀 Cloneable
 
+- 📟 [new-tailwind-app](https://github.com/msaaddev/new-tailwind-app) - Creates React.js, Next.js, Gatsby.js, Vue3, Laravel, and basic Tailwind CSS apps.
```

Comment:
This must be at the end of the emoji group. Per the guidelines: 
> Unless specified otherwise below, an item must be added to the bottom of its emoji group.